src/olympia/addons/models.py

```diff
@@ -708,23 +708,31 @@ def disable_all_files(cls, addons, reason):
         )
 
     def set_needs_human_review_on_latest_versions(
```

Comment:
This method has a return type that doesn't match the signature (in my internal mental model of how set/get methods are supposed to work).

Previously this method did what it says and then returned a boolean telling you if something changed. That makes sense.

But now it sets the flag and returns all the modified versions? Might be worht thinking about.